src/main/java/frc/robot/commands/AutonBalance.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.PIDCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.subsystems.DriveSubsystem;
+
+public class AutonBalance extends SequentialCommandGroup {
+
+  private class Balance extends PIDCommand {
+    DriveSubsystem driveSubsystem;
+
+    public Balance(DriveSubsystem driveSubsystem) {
+      super(new PIDController(0.3, 0, 0), driveSubsystem.pigeon2::getRoll, 0, output -> {
+        driveSubsystem.drive(new ChassisSpeeds(output * Constants.DRIVE_MAX_VELOCITY_METERS_PER_SECOND, 0,0));
+      }, driveSubsystem);
+      this.driveSubsystem = driveSubsystem;
+    }
+
+    @Override
+    public void execute() {
+        super.execute();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return Math.abs(m_controller.getPositionError()) < 1;
+    }
+    @Override
+    public void end(boolean interrupted) {
+        super.end(interrupted);
+        System.out.println("Ending Autonbalance");
+        driveSubsystem.drive(new ChassisSpeeds());
+    }
+  }
+
+  public AutonBalance(DriveSubsystem driveSubsystem) {
```

Comment:
Probably should add the option to pass a path as an argument since this is intended to be used with paths, and it seems a bit redundant to have a sequential command group that calls another sequential command group.

Reply:
I disagree. Parking would likely be the last thing that takes place, so why would it need to consume a path to park after it just completed following one?